runners/extensions-java/metrics/build.gradle

```diff
@@ -17,18 +17,20 @@
  */
 
 apply from: project(":").file("build_rules.gradle")
-applyJavaNature()
+applyJavaNature(failOnWarning: true)
 
 description = "Apache Beam :: Runners :: Extensions Java :: Metrics"
 ext.summary = "Beam Runners Extensions Metrics provides implementations of runners core metrics APIs."
 
 
 dependencies {
     compile library.java.guava
+    compileOnly library.java.findbugs_annotations
     shadow project(path: ":beam-sdks-java-core", configuration: "shadow")
     shadow library.java.jackson_databind
     shadow library.java.jackson_datatype_joda
-    shadow library.java.findbugs_annotations
+    shadow library.java.findbugs_jsr305
```

Comment:
Why is it needed as you did not add any findbugs annotation?

Reply:
Because otherwise error-prone does not pass.